processing file asmFiles/msi1.asm (1 of 2)
diff: memcpu.hex: No such file or directory
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
--- memsim.hex	2025-04-15 23:59:46.016511916 -0400
+++ memcpu.hex	2025-04-15 23:59:45.995511115 -0400
@@ -15,9 +15,9 @@
 :04000E0018B2A42F51
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
-:0400200000000002DA
+:0400200000000001DB
 :0400220000000001D9
-:0400240000000002D6
+:0400240000000001D7
 :0400260000000001D5
 :040080000000223723
 :04008100FC02021368
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
--- memsim.hex	2025-04-16 00:01:58.190557271 -0400
+++ memcpu.hex	2025-04-16 00:01:58.169556470 -0400
@@ -15,9 +15,9 @@
 :04000E0018B2A42F51
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
-:0400200000000002DA
+:0400200000000001DB
 :0400220000000001D9
-:0400240000000002D6
+:0400240000000001D7
 :0400260000000001D5
 :040080000000223723
 :04008100FC02021368
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
--- memsim.hex	2025-04-16 00:07:30.158229164 -0400
+++ memcpu.hex	2025-04-16 00:07:30.137228362 -0400
@@ -15,9 +15,9 @@
 :04000E0018B2A42F51
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
-:0400200000000002DA
+:0400200000000001DB
 :0400220000000001D9
-:0400240000000002D6
+:0400240000000001D7
 :0400260000000001D5
 :040080000000223723
 :04008100FC02021368
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:11:04.692418413 -0400
+++ memcpu.hex	2025-04-16 00:07:32.272309863 -0400
@@ -1,46 +1,240 @@
-:0400000000010137C3
-:04000100FFC1011327
-:04000200028000EF89
-:04000300FFFFFFFFFD
-:04000400100522AF12
-:04000500FE029EE376
-:0400060000100313D0
-:04000700186523AFA6
-:04000800FE0398E378
-:04000900000080670C
-:04000A0000052023AA
-:04000B00000080670A
-:04000C00FFC101131C
-:04000D00001120239B
-:04000E003000651346
-:04000F00FD5FF0EFB2
-:040010003040639386
-:040011000003A283C3
-:040012000022831332
-:040013000063A023C3
-:040014003000651340
-:04001500FD5FF0EFAC
-:040016000001208342
-:040017000041011390
-:0400180000008067FD
-:0400800000008137C4
-:04008100FFC10113A7
-:04008200008000EF0B
-:04008300FFFFFFFF7D
-:04008400FFC10113A4
-:040085000011202323
-:0400860030006513CE
-:04008700DF5FF0EF58
-:04008800304063930E
-:040089000003A2834B
-:04008A0000128313CA
-:04008B000063A0234B
-:04008C0030006513C8
-:04008D00DF5FF0EF52
-:04008E0000012083CA
-:04008F000041011318
-:040090000000806785
-:0400C1000000000338
-:041FFE000000020CD1
-:043FFE000000000CB3
+:04000000FFC06413C6
+:04000100FFC06113C8
+:04000200FFFF71B7D4
+:040003004031013354
+:04000400403404334D
+:04000500000071B7CF
+:040006000031013391
+:04000700003404338A
+:0400080040806613BB
+:0400090040402C0344
+:04000A0000106293ED
+:04000B00005C56B38C
+:04000C0000C064B319
+:04000D0000D0693383
+:04000E00210000EFDE
+:04000F0000106293E8
+:04001000005C52B38B
+:04001100405C06B396
+:040012000020631354
+:04001300006292B342
+:0400140040806613AF
+:040015000056063358
+:0400160000C069B30A
+:0400170000D06A3378
+:0400180000906633BB
+:04001900012066B3A9
+:04001A000130673317
+:04001B00014067B386
+:04001C00600062938B
+:04001D00FFC101130B
+:04001E00005120234A
+:04001F004000631327
+:040020000003238333
+:04002100FE038EE369
+:04002200208000EF4B
+:040023000041011384
+:04002400FFFFFFFFDC
+:04008000FFFFC1B706
+:04008100FFC0641345
+:04008200FFC0611347
+:0400830040310133D4
+:0400840040340433CD
+:04008500408066133E
+:0400860040402C03C7
+:040087000010629370
+:04008800005C56B30F
+:04008900002062935E
+:04008A00005692B3D7
+:04008B0000560633E2
+:04008C0040DC06B39B
+:04008D00014000EF3F
+:04008E004000629339
+:04008F0000106313E7
+:040090000062A02347
+:04009100FFFFFFFF6F
+:040092000040629335
+:0400930000206313D3
+:04009400006693333C
+:040095000062B23320
+:0400960002020A63F5
+:0400970000560FB34D
+:04009800000FAF03A3
+:0400990000CF8E63A3
+:04009A00FFCFA803E9
+:04009B00010F2233FC
+:04009C0000020863F3
+:04009D00010FA0238C
+:04009E00FFCF8F936E
+:04009F00FE9FF06F61
+:0400A00001EFA023A9
+:0400A1000042829304
+:0400A200FCDFF06F20
+:0400A3000000806772
+:0400A40000012283B2
+:0400A5000006966358
+:0400A6000007946358
+:0400A70007C0006F1F
+:0400A8000407806366
+:0400A90004068C635A
+:0400AA000006230326
+:0400AB0000072383A4
+:0400AC000073223388
+:0400AD0000020C63DE
+:0400AE000062A02329
+:0400AF0000428293F6
+:0400B00000460613ED
+:0400B100FFF686933D
+:0400B200FCDFF06F10
+:0400B3000072A02314
+:0400B40000428293F1
+:0400B50000470713E6
+:0400B600FFF7879336
+:0400B700FB9FF06F4C
+:0400B8000006230318
+:0400B9000062A0231E
+:0400BA0000428293EB
+:0400BB00FFF6869333
+:0400BC0000460613E1
+:0400BD000206826352
+:0400BE00FE9FF06F42
+:0400BF000007238390
+:0400C0000072A02307
+:0400C10000428293E4
+:0400C200FFF787932A
+:0400C30000470713D8
+:0400C400000784634A
+:0400C500FE9FF06F3B
+:0400C600000080674F
+:0401000000000001FA
+:0401010000000040BA
+:0401020000000001F8
+:0401030000000003F5
+:0401040000000004F3
+:0401050000000007EF
+:040106000000000BEA
+:040107000000000CE8
+:0401080000000015DE
+:0401090000000016DC
+:04010A0000000016DB
+:04010B0000000019D7
+:04010C000000001BD4
+:04010D000000001DD1
+:04010E0000000020CD
+:04010F0000000021CB
+:0401100000000029C2
+:0401110000000033B7
+:0401120000000036B3
+:0401130000000038B0
+:040114000000003DAA
+:040115000000003DA9
+:040116000000003EA7
+:0401170000000042A2
+:04011800000000479C
+:040119000000004B97
+:04011A000000004D94
+:04011B000000005090
+:04011C00000000518E
+:04011D00000000518D
+:04011E00000000538A
+:04011F000000005A82
+:040120000000006378
+:040121000000006476
+:0401220000000007D2
+:0401230000000007D1
+:0401240000000007D0
+:0401250000000008CE
+:0401260000000008CD
+:0401270000000008CC
+:0401280000000009CA
+:040129000000000AC8
+:04012A000000000FC2
+:04012B0000000010C0
+:04012C0000000011BE
+:04012D0000000017B7
+:04012E000000001AB3
+:04012F0000000020AC
+:0401300000000020AB
+:0401310000000023A7
+:0401320000000024A5
+:0401330000000026A2
+:0401340000000026A1
+:04013500000000289E
+:040136000000002F96
+:040137000000003094
+:040138000000003291
+:040139000000004181
+:04013A00000000427F
+:04013B00000000437D
+:04013C00000000447B
+:04013D000000005965
+:04013E000000005B62
+:04013F000000005C60
+:040140000000005C5F
+:040141000000006456
+:04018000000000017A
+:040181000000000377
+:040182000000000475
+:040183000000000771
+:040184000000000770
+:04018500000000076F
+:04018600000000076E
+:04018700000000086C
+:04018800000000086B
+:04018900000000086A
+:04018A000000000968
+:04018B000000000A66
+:04018C000000000B64
+:04018D000000000C62
+:04018E000000000F5E
+:04018F00000000105C
+:04019000000000115A
+:040191000000001555
+:040192000000001653
+:040193000000001652
+:040194000000001750
+:04019500000000194D
+:040196000000001A4B
+:040197000000001B49
+:040198000000001D46
+:040199000000002042
+:04019A000000002041
+:04019B000000002040
+:04019C00000000213E
+:04019D00000000233B
+:04019E000000002439
+:04019F000000002636
+:0401A0000000002635
+:0401A1000000002832
+:0401A2000000002930
+:0401A3000000002F29
+:0401A4000000003027
+:0401A5000000003224
+:0401A6000000003322
+:0401A700000000361E
+:0401A800000000381B
+:0401A9000000003D15
+:0401AA000000003D14
+:0401AB000000003E12
+:0401AC00000000410E
+:0401AD00000000420C
+:0401AE00000000420B
+:0401AF000000004309
+:0401B0000000004407
+:0401B1000000004703
+:0401B2000000004BFE
+:0401B3000000004DFB
+:0401B40000000050F7
+:0401B50000000051F5
+:0401B60000000051F4
+:0401B70000000053F1
+:0401B80000000059EA
+:0401B9000000005AE8
+:0401BA000000005BE6
+:0401BB000000005CE4
+:0401BC000000005CE3
+:0401BD0000000063DB
+:0401BE0000000064D9
+:0401BF0000000064D8
+:043FFE0000000600B9
 :00000001FF
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:11:33.316511138 -0400
+++ memcpu.hex	2025-04-16 00:11:33.291510184 -0400
@@ -40,7 +40,7 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C100000000013A
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:15:58.314627448 -0400
+++ memcpu.hex	2025-04-16 00:15:58.289626493 -0400
@@ -40,7 +40,7 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C100000000013A
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2025-04-16 00:19:59.204823432 -0400
+++ memcpu.hex	2025-04-16 00:19:59.182822592 -0400
@@ -1,40 +1,26 @@
-:04000000FFC06413C6
-:04000100FFC06113C8
-:04000200FFFF71B7D4
-:040003004031013354
-:04000400403404334D
+:0400040000000008F0
 :04000500000071B7CF
-:040006000031013391
-:04000700003404338A
-:0400080040806613BB
-:0400090040402C0344
-:04000A0000106293ED
-:04000B00005C56B38C
-:04000C0000C064B319
-:04000D0000D0693383
-:04000E00210000EFDE
-:04000F0000106293E8
+:0400070000000020D5
+:04000C0000000001EF
+:04000E0000000004EA
 :04001000005C52B38B
 :04001100405C06B396
 :040012000020631354
 :04001300006292B342
-:0400140040806613AF
-:040015000056063358
-:0400160000C069B30A
-:0400170000D06A3378
-:0400180000906633BB
-:04001900012066B3A9
-:04001A000130673317
-:04001B00014067B386
+:0400150000000017D0
+:040016000000006482
+:0400190000000008DB
+:04001A0000000015CD
 :04001C00600062938B
-:04001D00FFC101130B
-:04001E00005120234A
-:04001F004000631327
+:04001D0000000004DB
+:04001F0000000007D6
 :040020000003238333
 :04002100FE038EE369
 :04002200208000EF4B
 :040023000041011384
-:04002400FFFFFFFFDC
+:0400240000000008D0
+:040025000000000FC8
+:0400310000000021AA
 :04008000FFFFC1B706
 :04008100FFC0641345
 :04008200FFC0611347
@@ -106,135 +92,53 @@
 :0400C400000784634A
 :0400C500FE9FF06F3B
 :0400C600000080674F
-:0401000000000001FA
-:0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
-:0401090000000016DC
+:04010300FFC06113C5
+:0401050000000019DD
+:0401060000000050A5
+:0401070000000029CB
 :04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
-:040120000000006378
-:040121000000006476
-:0401220000000007D2
-:0401230000000007D1
-:0401240000000007D0
-:0401250000000008CE
-:0401260000000008CD
-:0401270000000008CC
-:0401280000000009CA
-:040129000000000AC8
-:04012A000000000FC2
-:04012B0000000010C0
-:04012C0000000011BE
-:04012D0000000017B7
-:04012E000000001AB3
-:04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
-:0401320000000024A5
-:0401330000000026A2
-:0401340000000026A1
-:04013500000000289E
-:040136000000002F96
-:040137000000003094
-:040138000000003291
-:040139000000004181
-:04013A00000000427F
-:04013B00000000437D
-:04013C00000000447B
-:04013D000000005965
-:04013E000000005B62
-:04013F000000005C60
-:040140000000005C5F
-:040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
-:040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
-:0401BF0000000064D8
-:043FFE0000000600B9
+:04010B000000003EB2
+:04010C000000004BA4
+:04010D0000000047A7
+:04010E00000000539A
+:04010F00000000519B
+:040110000000004D9E
+:0401110000000016D4
+:0401130000000007E1
+:0401140000000007E0
+:0401150000000021C5
+:0401160000000029BC
+:040117000000003DA7
+:04011800000000647F
+:0401190000000042A0
+:04011B0000000021BF
+:04011D000000001AC4
+:04011F0000000008D4
+:0401200000000038A3
+:040121000000005189
+:0401230000000008D0
+:040124000000004295
+:040125000000006472
+:0401270000000029AB
+:0401280000000016BD
+:0401290000000015BD
+:04012B000000003E92
+:04012D000000004787
+:04012F00000000517B
+:040130000000004487
+:0401310000000020AA
+:0401330000000008C0
+:040134000000000FB8
+:0401350000000017AF
+:040136000000001AAB
+:0401370000000020A4
+:0401390000000008BA
+:04013A00000000239E
+:04013B00000000249C
+:04013D0000000010AE
+:04013F0000000007B5
+:0401400000000007B4
+:040141000000000AB0
+:041F01000000004498
+:042001000000000AD1
 :00000001FF
processing file asmFiles/dual.llsc.asm (1 of 2)
--- memsim.hex	2025-04-16 00:20:34.440168543 -0400
+++ memcpu.hex	2025-04-16 00:20:34.417167665 -0400
@@ -15,10 +15,6 @@
 :04000E0018B2A42F51
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
-:0400200000000002DA
-:0400220000000001D9
-:0400240000000002D6
-:0400260000000001D5
 :040080000000223723
 :04008100FC02021368
 :0400820009006513F9
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2025-04-16 00:20:36.719255547 -0400
+++ memcpu.hex	2025-04-16 00:20:36.696254669 -0400
@@ -108,133 +108,131 @@
 :0400C600000080674F
 :0401000000000001FA
 :0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
+:0401020000000003F6
+:040103000000005A9E
+:040104000000005A9D
+:0401050000000033C3
+:0401060000000033C2
+:0401070000000050A4
+:0401080000000050A3
 :0401090000000016DC
 :04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
+:04010B000000000CE4
+:04010C000000000CE3
+:04010D000000004BA3
+:04010E000000004BA2
+:04010F000000005399
+:040110000000005398
+:040111000000005A90
+:040112000000005A8F
+:040113000000005A8E
+:040114000000005A8D
+:040115000000005A8C
+:040116000000006382
+:040117000000006381
+:040118000000006380
+:04011900000000637F
+:04011A00000000637E
+:04011B00000000637D
+:04011C00000000637C
+:04011D00000000637B
+:04011E00000000637A
+:04011F000000006379
 :040120000000006378
-:040121000000006476
-:0401220000000007D2
-:0401230000000007D1
-:0401240000000007D0
-:0401250000000008CE
-:0401260000000008CD
-:0401270000000008CC
-:0401280000000009CA
-:040129000000000AC8
-:04012A000000000FC2
-:04012B0000000010C0
-:04012C0000000011BE
-:04012D0000000017B7
-:04012E000000001AB3
-:04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
-:0401320000000024A5
-:0401330000000026A2
-:0401340000000026A1
-:04013500000000289E
-:040136000000002F96
-:040137000000003094
-:040138000000003291
-:040139000000004181
-:04013A00000000427F
-:04013B00000000437D
-:04013C00000000447B
-:04013D000000005965
-:04013E000000005B62
-:04013F000000005C60
+:0401210000000003D7
+:0401220000000030A9
+:040124000000004295
+:040125000000004294
+:040126000000000FC6
+:040127000000000FC5
+:040128000000004192
+:0401290000000020B2
+:04012A0000000009C8
+:04012B000000002FA1
+:04012C000000005976
+:04012D0000000011BD
+:04012E0000000007C6
+:04012F0000000023A9
+:040130000000006467
+:0401310000000020AA
+:040132000000006465
+:0401330000000007C1
+:040134000000006463
+:040135000000006462
+:040136000000006461
+:040137000000006460
+:04013800000000645F
+:0401390000000008BA
+:04013A00000000645D
+:04013B00000000269A
+:04013C00000000645B
+:04013D000000002698
+:04013E000000006459
+:04013F00000000328A
 :040140000000005C5F
 :040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
-:040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
+:040180000000000378
+:04018100000000304A
+:040183000000004236
+:040184000000004235
+:040185000000000F67
+:040186000000000F66
+:040187000000004133
+:040188000000002053
+:040189000000000969
+:04018A000000002F42
+:04018B000000005917
+:04018C00000000115E
+:04018D000000000767
+:04018E00000000234A
+:04018F000000005A12
+:040190000000005A11
+:040191000000003337
+:040192000000003336
+:040193000000005018
+:040194000000005017
+:040195000000001650
+:04019600000000164F
+:040197000000000C58
+:040198000000000C57
+:040199000000004B17
+:04019A000000004B16
+:04019B00000000530D
+:04019C00000000530C
+:04019D000000005A04
+:04019E000000005A03
+:04019F000000005A02
+:0401A0000000005A01
+:0401A1000000005A00
+:0401A20000000063F6
+:0401A30000000063F5
+:0401A40000000063F4
+:0401A50000000063F3
+:0401A60000000063F2
+:0401A70000000063F1
+:0401A80000000063F0
+:0401A90000000063EF
+:0401AA0000000063EE
+:0401AB0000000063ED
+:0401AC0000000063EC
+:0401AD00000000034B
+:0401AE0000000064E9
+:0401AF00000000202C
+:0401B00000000064E7
+:0401B1000000000743
+:0401B20000000064E5
+:0401B30000000064E4
+:0401B40000000064E3
+:0401B50000000064E2
+:0401B60000000064E1
+:0401B700000000083C
+:0401B80000000064DF
+:0401B900000000261C
+:0401BA0000000064DD
+:0401BB00000000261A
+:0401BC0000000064DB
+:0401BD00000000320C
+:0401BE000000005CE1
 :0401BF0000000064D8
 :043FFE0000000600B9
 :00000001FF
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:21:33.628428057 -0400
+++ memcpu.hex	2025-04-16 00:21:33.607427255 -0400
@@ -40,7 +40,7 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C100000000013A
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
--- memsim.hex	2025-04-16 00:23:48.809588600 -0400
+++ memcpu.hex	2025-04-16 00:23:48.789587836 -0400
@@ -19,7 +19,7 @@
 :0400860000732023C0
 :04008700FFFFFFFF79
 :0401000000000001FA
-:04010200DEADBEEFC1
+:04010200DEAD00006E
 :040103000000DEAD6D
 :040105000000BEEF49
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:29:31.922686766 -0400
+++ memcpu.hex	2025-04-16 00:29:31.902686003 -0400
@@ -40,7 +40,8 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C000000000013B
+:0400C1000000000239
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/dual.llsc.asm (1 of 2)
--- memsim.hex	2025-04-16 00:29:36.782872294 -0400
+++ memcpu.hex	2025-04-16 00:29:36.763871568 -0400
@@ -15,10 +15,6 @@
 :04000E0018B2A42F51
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
-:0400200000000002DA
-:0400220000000001D9
-:0400240000000002D6
-:0400260000000001D5
 :040080000000223723
 :04008100FC02021368
 :0400820009006513F9
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2025-04-16 00:29:39.145962500 -0400
+++ memcpu.hex	2025-04-16 00:29:39.123961660 -0400
@@ -108,133 +108,131 @@
 :0400C600000080674F
 :0401000000000001FA
 :0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
+:0401020000000003F6
+:040103000000005A9E
+:040104000000005A9D
+:0401050000000033C3
+:0401060000000033C2
+:0401070000000050A4
+:0401080000000050A3
 :0401090000000016DC
 :04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
+:04010B000000000CE4
+:04010C000000000CE3
+:04010D000000004BA3
+:04010E000000004BA2
+:04010F000000005399
+:040110000000005398
+:040111000000005A90
+:040112000000005A8F
+:040113000000005A8E
+:040114000000005A8D
+:040115000000005A8C
+:040116000000006382
+:040117000000006381
+:040118000000006380
+:04011900000000637F
+:04011A00000000637E
+:04011B00000000637D
+:04011C00000000637C
+:04011D00000000637B
+:04011E00000000637A
+:04011F000000006379
 :040120000000006378
-:040121000000006476
-:0401220000000007D2
-:0401230000000007D1
-:0401240000000007D0
-:0401250000000008CE
-:0401260000000008CD
-:0401270000000008CC
-:0401280000000009CA
-:040129000000000AC8
-:04012A000000000FC2
-:04012B0000000010C0
-:04012C0000000011BE
-:04012D0000000017B7
-:04012E000000001AB3
-:04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
-:0401320000000024A5
-:0401330000000026A2
-:0401340000000026A1
-:04013500000000289E
-:040136000000002F96
-:040137000000003094
-:040138000000003291
-:040139000000004181
-:04013A00000000427F
-:04013B00000000437D
-:04013C00000000447B
-:04013D000000005965
-:04013E000000005B62
-:04013F000000005C60
+:0401210000000003D7
+:0401220000000030A9
+:040124000000004295
+:040125000000004294
+:040126000000000FC6
+:040127000000000FC5
+:040128000000004192
+:0401290000000020B2
+:04012A0000000009C8
+:04012B000000002FA1
+:04012C000000005976
+:04012D0000000011BD
+:04012E0000000007C6
+:04012F0000000023A9
+:040130000000006467
+:0401310000000020AA
+:040132000000006465
+:0401330000000007C1
+:040134000000006463
+:040135000000006462
+:040136000000006461
+:040137000000006460
+:04013800000000645F
+:0401390000000008BA
+:04013A00000000645D
+:04013B00000000269A
+:04013C00000000645B
+:04013D000000002698
+:04013E000000006459
+:04013F00000000328A
 :040140000000005C5F
 :040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
-:040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
+:040180000000000378
+:04018100000000304A
+:040183000000004236
+:040184000000004235
+:040185000000000F67
+:040186000000000F66
+:040187000000004133
+:040188000000002053
+:040189000000000969
+:04018A000000002F42
+:04018B000000005917
+:04018C00000000115E
+:04018D000000000767
+:04018E00000000234A
+:04018F000000005A12
+:040190000000005A11
+:040191000000003337
+:040192000000003336
+:040193000000005018
+:040194000000005017
+:040195000000001650
+:04019600000000164F
+:040197000000000C58
+:040198000000000C57
+:040199000000004B17
+:04019A000000004B16
+:04019B00000000530D
+:04019C00000000530C
+:04019D000000005A04
+:04019E000000005A03
+:04019F000000005A02
+:0401A0000000005A01
+:0401A1000000005A00
+:0401A20000000063F6
+:0401A30000000063F5
+:0401A40000000063F4
+:0401A50000000063F3
+:0401A60000000063F2
+:0401A70000000063F1
+:0401A80000000063F0
+:0401A90000000063EF
+:0401AA0000000063EE
+:0401AB0000000063ED
+:0401AC0000000063EC
+:0401AD00000000034B
+:0401AE0000000064E9
+:0401AF00000000202C
+:0401B00000000064E7
+:0401B1000000000743
+:0401B20000000064E5
+:0401B30000000064E4
+:0401B40000000064E3
+:0401B50000000064E2
+:0401B60000000064E1
+:0401B700000000083C
+:0401B80000000064DF
+:0401B900000000261C
+:0401BA0000000064DD
+:0401BB00000000261A
+:0401BC0000000064DB
+:0401BD00000000320C
+:0401BE000000005CE1
 :0401BF0000000064D8
 :043FFE0000000600B9
 :00000001FF
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
--- memsim.hex	2025-04-16 00:33:36.268014145 -0400
+++ memcpu.hex	2025-04-16 00:33:36.245013268 -0400
@@ -1,5 +1,4 @@
 :0400000000002237A3
-:04000100FC020213E8
 :04000200080065137A
 :04000300000525834C
 :0400040000158593CB
@@ -16,9 +15,6 @@
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
 :0400200000000002DA
-:0400220000000001D9
-:0400240000000002D6
-:0400260000000001D5
 :040080000000223723
 :04008100FC02021368
 :0400820009006513F9
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2025-04-16 00:33:38.218088586 -0400
+++ memcpu.hex	2025-04-16 00:33:38.196087746 -0400
@@ -1,5 +1,3 @@
-:04000000FFC06413C6
-:04000100FFC06113C8
 :04000200FFFF71B7D4
 :040003004031013354
 :04000400403404334D
@@ -16,14 +14,11 @@
 :04000F0000106293E8
 :04001000005C52B38B
 :04001100405C06B396
-:040012000020631354
-:04001300006292B342
-:0400140040806613AF
-:040015000056063358
+:0400130000000007E2
+:040015000000005C8B
 :0400160000C069B30A
 :0400170000D06A3378
-:0400180000906633BB
-:04001900012066B3A9
+:0400190000000042A1
 :04001A000130673317
 :04001B00014067B386
 :04001C00600062938B
@@ -108,133 +103,61 @@
 :0400C600000080674F
 :0401000000000001FA
 :0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
-:0401090000000016DC
-:04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
-:040120000000006378
-:040121000000006476
-:0401220000000007D2
-:0401230000000007D1
-:0401240000000007D0
-:0401250000000008CE
-:0401260000000008CD
-:0401270000000008CC
-:0401280000000009CA
-:040129000000000AC8
-:04012A000000000FC2
-:04012B0000000010C0
-:04012C0000000011BE
-:04012D0000000017B7
-:04012E000000001AB3
-:04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
-:0401320000000024A5
-:0401330000000026A2
-:0401340000000026A1
-:04013500000000289E
-:040136000000002F96
-:040137000000003094
-:040138000000003291
-:040139000000004181
-:04013A00000000427F
-:04013B00000000437D
-:04013C00000000447B
-:04013D000000005965
-:04013E000000005B62
-:04013F000000005C60
-:040140000000005C5F
-:040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
+:0401040000000033C4
+:0401050000000019DD
+:0401060000000050A5
+:0401070000000029CB
+:0401090000000020D2
+:04010A000000000CE5
+:04010B000000003EB2
+:04010D000000003DB1
+:0401110000000016D4
+:04011B00000000518F
+:04011C0000000001DE
+:04011D0000000036A8
+:04011E00000000429B
+:04011F000000004399
+:0401200000000038A3
+:040121000000004397
+:0401220000000030A9
+:0401230000000008D0
+:0401240000000033A4
+:0401250000000019BD
+:040126000000000FC6
+:040127000000005C78
+:040128000000004192
+:0401290000000020B2
+:04012A0000000009C8
+:04012B000000002FA1
+:04012D0000000011BD
+:04012E0000000007C6
+:04012F0000000023A9
+:040130000000004487
+:0401310000000020AA
+:04013700000000289C
+:0401380000000007BC
+:0401390000000008BA
+:04013A0000000010B1
+:04013B0000000011AF
+:04013C000000001AA5
+:04013D000000002F8F
+:04013F00000000328A
+:040140000000004378
+:040141000000005C5E
+:040182000000003346
 :040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
-:0401BF0000000064D8
-:043FFE0000000600B9
+:040196000000005A0B
+:0401A2000000003E1B
+:0401A6000000002035
+:0401B20000000051F8
+:0401B3000000000147
+:0401B4000000003611
+:0401B5000000004204
+:0401B6000000004302
+:0401B700000000380C
+:0401B8000000004300
+:0401B90000000051F1
+:0401BA000000000140
+:0401BC0000000042FD
+:0401BE000000003805
 :00000001FF
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:33:44.141314692 -0400
+++ memcpu.hex	2025-04-16 00:33:44.121313928 -0400
@@ -6,8 +6,6 @@
 :04000500FE029EE376
 :0400060000100313D0
 :04000700186523AFA6
-:04000800FE0398E378
-:04000900000080670C
 :04000A0000052023AA
 :04000B00000080670A
 :04000C00FFC101131C
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:34:00.961956786 -0400
+++ memcpu.hex	2025-04-16 00:34:00.939955946 -0400
@@ -40,7 +40,7 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C100000000013A
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:36:53.473542056 -0400
+++ memcpu.hex	2025-04-16 00:36:53.454541331 -0400
@@ -40,7 +40,7 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C100000000013A
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/palgorithm.asm (1 of 1)
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2025-04-16 00:44:46.603602813 -0400
+++ memcpu.hex	2025-04-16 00:44:46.579601897 -0400
@@ -40,7 +40,7 @@
 :04008E0000012083CA
 :04008F000041011318
 :040090000000806785
-:0400C1000000000338
+:0400C100000000013A
 :041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/test.lrsc.asm (1 of 1)
--- memsim.hex	2025-04-16 00:45:15.714714071 -0400
+++ memcpu.hex	2025-04-16 00:45:15.691713193 -0400
@@ -38,6 +38,7 @@
 :04008D0030006513C7
 :04008E00DF1FF0EF91
 :04008F00FFFFFFFF71
-:0400C1000000000A31
+:0400C1000000000536
+:041FFE000000020CD1
 :043FFE000000000CB3
 :00000001FF
processing file asmFiles/stom.asm (1 of 1)
--- memsim.hex	2025-04-16 00:50:34.211872475 -0400
+++ memcpu.hex	2025-04-16 00:50:34.193871788 -0400
@@ -3,7 +3,6 @@
 :0400020000158593CD
 :0400030000B5202301
 :04000400FFFFFFFFFC
-:0400200000000001DB
 :040080000000001369
 :040081000000001368
 :040082000000001367
processing file asmFiles/stom.asm (1 of 1)
processing file asmFiles/stom.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
